podcasts.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/firebase/abseil-cpp-SwiftPM.git",
         "state": {
           "branch": null,
-          "revision": "d302de612e3d57c6f4afaf087da18fba8eac72a7",
-          "version": "0.20220203.1"
+          "revision": "583de9bd60f66b40e78d08599cc92036c2e7e4e1",
```

Comment:
I believe we can undo the changes on this file since they are not related to this task. But this reminds me that it would be good to update our dependencies.

Reply:
Updated.